migrations/12_SCHEMES_GenericScheme.js

```diff
@@ -20,15 +20,22 @@ module.exports = async function (deployer) { // eslint-disable-line no-unused-va
   console.log('Configure GenericScheme')
 
   const {
-    genesisProtocolParamsHash,
-    genesisProtocolAddress
-  } = await getGenesisProtocolData()
+    paramsHash: genesisProtocolParamsHash,
+    address: genesisProtocolAddress
+  } = await genesisProtocolHelper.getGenesisProtocolData('dutchX')
 
   // DutchX address
-  const dutchXContractAddress = await getDXContractAddress('DutchExchangeProxy')
+  let dutchXContractAddress
+  if (process.env.USE_MOCK_DX) {
+    const Wallet = artifacts.require('Wallet')
+    const WalletDeployed = await Wallet.deployed()
+    dutchXContractAddress = WalletDeployed.address
```

Comment:
Deploying Wallet as Mock DX

Reply:
do you have a separate script to transfer ownership to the dao ?

Reply:
Ownership is transferred in migration `15`, ignore that it was commented out. Bugged out in dry-run, that's all